internal/host/connectivitycheckconvertor.go

```diff
@@ -13,6 +16,9 @@ import (
 func convertHostsToConnectivityCheckParams(currentHostId *strfmt.UUID, hosts []*models.Host, connectivityValidator connectivity.Validator) (string, error) {
 	var connectivityCheckHosts models.ConnectivityCheckParams
 	for i := range hosts {
+		if funk.ContainsString([]string{models.HostStatusDiscovering, models.HostStatusDisabled}, swag.StringValue(hosts[i].Status)) {
```

Comment:
why discovering? I think it's by mistake and you want to write here disconnected, right?

Reply:
Actually I meant discovering, but I will add disconnected as well - since discovering host is a host that doesn't have inventory yet, so we should not check it.

Reply:
ok, please add a comment why we don't need to check connectivity check in those states

Reply:
OK